assets/helpers/stripeCheckout/stripeCheckout.js

```diff
@@ -52,12 +52,20 @@ export const setupStripeCheckout = (
   });
 });
 
-export const openDialogBox = (amount: number, email: string) => {
-  if (stripeHandler) {
+export const openDialogBox = (
+  amount: number,
+  email: string,
+  postDeployTestCallback: Function,
+  isPostDeploymentTestUser: boolean,
+) => {
+  if (stripeHandler && !isPostDeploymentTestUser) {
     stripeHandler.open({
       // Must be passed in pence.
       amount: amount * 100,
       email,
     });
+  } else if (stripeHandler && isPostDeploymentTestUser) {
+    const testTokenId = 'tok_visa';
```

Comment:
This is publicly available in the Stripe docs: https://stripe.com/docs/testing#cards